packages/amplify_datastore/example/lib/models/Blog.dart

```diff
@@ -120,6 +164,8 @@ class Blog extends Model {
       fieldName: "posts",
       fieldType: ModelFieldType(ModelFieldTypeEnum.model,
           ofModelName: (Post).toString()));
+  static final QueryField CREATEDAT = QueryField(fieldName: "createdAt");
+  static final QueryField UPDATEDAT = QueryField(fieldName: "updatedAt");
```

Comment:
I may be wrong I think `createdAt` and `updatedAt` should not be query fields.

Reply:
Update: compared native Android and iOS models, `createdAt` and `updatedAt` are not query fields.

Reply:
QueryFields removed